Representor/HTTP/Adapters/HTTPSirenAdapter.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  SirenAdapter.swift
```

Comment:
This header comment wasn't updated to the new filename when it was moved.

But it's HTTPSirenAdapter instead of SirenAdapter because it uses the HTTP Transition for Siren actions and is a HTTP specific format.